src/cli/java/org/commcare/util/screen/EntityScreen.java

```diff
@@ -241,7 +241,27 @@ protected boolean executePendingAction(CommCareSession session) {
      */
     public void updateSelection(String input, @Nullable String[] selectedValues) throws CommCareSessionException {
```

Comment:
is this ever called now?

Reply:
yes, this is still called for [normal FP navigation](https://github.com/dimagi/commcare-core/blob/entityScreenRefactor/src/cli/java/org/commcare/util/screen/EntityListSubscreen.java#L248), while the new updateSelection with TreeReferences is invoked for CLI only.